python/shotgun_model/shotgun_hierarchy_model.py

```diff
@@ -22,6 +22,7 @@
 from .shotgun_query_model import ShotgunQueryModel
 from .data_handler_nav import ShotgunNavDataHandler
 from .util import sanitize_for_qt_model
+from ..utils import safe_delete_later
```

Comment:
You may want to try to build the sphinx docs to make sure this works. I think you may have to do the toolkit-style import:

```python
utils = sgtk.platform.current_bundle().import_module("utils")
```